pkg/deploy/generator/resources.go

```diff
@@ -28,6 +28,10 @@ import (
 
 const (
 	tenantIDHack = "13805ec3-a223-47ad-ad65-8b2baf92c0fb"
+	//aro-billing-operator SP Id, from https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/7547877/
+	billingSPID = "970792b5-7720-4bf5-a335-f15e97c7e25a"
```

Comment:
Let's make this parameter in RP-Config so we can change in case of billing migration happens
Sorry for late review;/